ibm_catalog.json

```
@@ -493,7 +493,7 @@
             "features": [
               {
                 "title": " ",
-                "description": "Configured to use IBM secure by default standards, but can be edited to fit your use case."
+                "description": "Configuring IBM Cloud Activity Tracker Event Routing target to an object storage bucket and cloud logs instance."
```

Hmm, the wording here is not quite accurate. How about:
```suggestion
                "description": "Configures the Activity Tracker Event Routing service to route auditing events to an Object Storage bucket and Cloud Logs target"
```